src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
```suggestion
  for (const action of actions) {
    const { type, extraData, keysToRemove } = actions;
    
    switch (type) {
```
You can also destructure all the needed props to make your code a bit cleaner)